File: HERO C#/PositionClosedLoopAuxiliary[FeedForward]/Framework/Gains.cs
using System;
using Microsoft.SPOT;

namespace PositionClosedLoopAuxiliary.Framework
{
    public class Gains
    {
        public float kP;
        public float kI;
        public float kD;
        public float kF;
        public float kIzone;
        public float kPeakOutput;

        public Gains(float _kP, float _kI, float _kD, float _kF, float _kIzone, float _kPeakOutput)
        {
            kP = _kP;
            kI = _kI;
            kD = _kD;
            kF = _kF;
            kIzone = _kIzone;
            kPeakOutput = _kPeakOutput;
        }
    }
}

File: Java/_TalonFX & Falcon 500 Examples/MotionProfile_Simple/src/main/java/frc/robot/Gains.java
package frc.robot;

/**
 *  Class that organizes gains used when assigning values to slots
 */
public class Gains {
	public final double kP;
	public final double kI;
	public final double kD;
	public final double kF;
	public final double kIzone;
	public final double kPeakOutput;
	
	public Gains(double _kP, double _kI, double _kD, double _kF, double _kIzone, double _kPeakOutput){
		kP = _kP;
		kI = _kI;
		kD = _kD;
		kF = _kF;
		kIzone = _kIzone;
		kPeakOutput = _kPeakOutput;
	}
}
